src/04 - Web services.md

```diff
@@ -56,6 +58,157 @@ Note that all services must remain backwards-compatible even when their version
 - Dates must be encoded as **ISO 8601** strings prior to sending them to the web services.
 - Dates are returned by the web services as **ISO 8601** formatted strings.
 
+# Code list management web services
+
+Address:
+```
+https://{server}:{port}/rms/api/public/noark5/v1/code-lists
+```
+
+Note that the values of all code lists have the same base fields (*code*, *name*, *description*), but some values have extra fields.
```

Comment:
I like this approach much better too. I would just suggest a slight rewording (if you agree with it):
```
Special note: All code list values have the same base fields: *code*, *name*, and *description*. Certain code list values, however, also support additional fields. They are described below.

The *skjerming* ...
```

Reply:
Changed.